src/app/store/selectors/VehicleTechRecordModel.selectors.spec.ts

```diff
@@ -0,0 +1,38 @@
+import {
```

Comment:
Please exclude this spec file in your commit. see latest 10202 for how it is done. The spec file is new so i wont worry about it in 10278

Reply:
I also refactored it using projector, I will push it like this then you can decide what to keep.